Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLink.cs

```diff
@@ -21,10 +41,30 @@ public class AudioLink : UdonSharpBehaviour
 
     using UnityEngine.Rendering;
 
+        public enum MediaPlaying
```

Comment:
These enums really shouldn't be duplicated. You should move them such that they are compiled into both code paths.

Reply:
Lmk if you need help with that

Reply:
Is there some other way I can do that?
When I tried it either complained about the enum coming before the using statements
Or complained about being embedded in a class

Reply:
this still needs to be addressed